types/account.d.ts

```diff
@@ -1,12 +1,19 @@
 import type { SerializedWatchHistoryItem } from './watchHistory';
 import type { SerializedFavorite } from './favorite';
 
+import type { Config } from '#types/Config';
+
 export type AuthData = {
   jwt: string;
   customerToken: string;
   refreshToken: string;
 };
 
+export type InPlayerAuthData = {
```

Comment:
It probably makes sense to split these into 3 files (generic account types, cleeng, and inplayer)